### PR TITLE
Adopt AniLibria.tv new page layout

### PR DESCRIPTION
**Pull request by alexlitvinenko**
#29

### CODE REVIEW

**Comment by idlesign** [comment]
Thank you, I'll try to check in a day or two.

File: torrt/rpc/deluge.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def method_get_torrents(self, hashes=None):
         return result['torrents']
 
     def method_add_torrent(self, torrent, download_to=None):
-        torrent_dump = base64encode(torrent).decode('utf-8')
+        torrent_dump = base64encode(torrent)
```
Comment (idlesign) [inline torrt/rpc/deluge.py]: That'll break thins in py2.

File: tests/test_utils.py [hunk 2]
```diff
@@ -16,3 +16,15 @@ def test_base64encode_bytes():
 
     assert isinstance(encoded_string, str)
     assert encoded_string == 'dGhpcyBpcyBieXRlcyB0byBlbmNvZGU=\n'
+
+
+def test_iter_notifiers_stop_iteration():
```
Comment (idlesign) [inline tests/test_utils.py]: Please do not pile unrelated changes into one pull request, but make several.
Let that be the request only about AniLibria. Py3 related requests are welcome as separate.

File: torrt/trackers/anilibria.py [hunk 3]
```diff
@@ -25,14 +25,14 @@ def get_download_link(self, url):
         page_soup = self.get_response(url, as_soup=True)
 
         available_qualities = {}
-        divs = page_soup.select('div.download-torrent')
-        for div in divs:
-            quality_span = div.select('div.torrent-first-col > span')
-            quality_str = quality_span[0].text.strip()
+        rows = page_soup.select('#publicTorrentTable tr')
```
Comment (idlesign) [inline torrt/trackers/anilibria.py]: I've just added a convinience to automatically test torent extraction https://github.com/idlesign/torrt/commit/b6d83e687056c62fe8dfea672efac98509397f5d

You can add the following or something to test your code automatically:
```
    test_urls = [
        'https://www.anilibria.tv/release/black-clover.html',
    ]
```

File: tests/test_utils.py (hunk 2 above)
Comment (alexlitvinenko) [reply tests/test_utils.py]: Sorry for that, you are definitely right.

File: torrt/rpc/deluge.py (hunk 1 above)
Comment (alexlitvinenko) [reply torrt/rpc/deluge.py]: I ran add_torrent_from_url() with tox - and there were no compatibility issues found. Also `base64encode(torrent)`[ is referenced](https://github.com/idlesign/torrt/blob/b6d83e687056c62fe8dfea672efac98509397f5d/torrt/rpc/transmission.py#L91) in TransmissionRPC without `decode('utf-8')` . Can you cover it with test to point out the problem? Maybe it makes sense to move `decode('utf-8')` into `base64encode(torrent)`.

**Pull request closed without merging**